runtime/protocol/http-client-engines/test-suite/common/test/aws/smithy/kotlin/runtime/http/test/ConnectionTest.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.smithy.kotlin.runtime.http.test
+
+// TODO Finish once we have HTTP engine support for client certificates
```

Comment:
Add a note to the TLS configuration issue to finish this.

Reply:
Added note to #820.